integration

```diff
@@ -332,6 +332,7 @@
   "finity69x2/nws_alerts",
   "firstof9/ha-openei",
   "FL550/dwd_weather",
+  "flexopus/flexopus-hass-sensor",
```

Comment:
Repository seems to have gone stale since October last year; @sebinemeth, are you still associated with this org?

Reply:
Yes, I am. The HA integration is not the main development focus of the company but it is maintained.

Reply:
Development priorities are constantly being aligned, especially when we have to wait such a long time for a review. Our integration is compatible with the newer versions of HA, and is constantly being tested, it is not a draft. Please do not block the publishing process because you think we are not developing actively enough.